### ISSUE TITLE
vscode-lit-plugin ignores `css.customData`

### ISSUE THREAD

**Issue by jimbojw** [runem/lit-analyzer#373]
According to the `vscode-css-languageservice` documentation, it supports configuration via the [`css.customData`](https://github.com/Microsoft/vscode-css-languageservice/blob/main/docs/customData.md) setting.

However, the `vscode-lit-plugin` does not appear to honor these settings when analyzing `css` templates. For example, it should be possible to define a custom `@` rule, like so:

`.vscode/settings.json`:

```json
{
  "css.customData": ["./.vscode/css-data.json"]
}
```

`.vscode/css-data.json` (attempting to work around #323):

```json
{
  "version": 1.1,
  "atDirectives": [
    {
      "name": "@container",
      "description": "https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries"
    }
  ]
}
```

Example TypeScript code:

```ts
import { css } from "lit";

export const EXAMPLE_CSS = css `
  @container (width < 600px) {
    /* ... */
  }
`;
```

Expected: No warnings.

Actual: Problems view reads `Unknown at rule @container lit-plugin(no-invalid-css)`.

**Comment by DominoPivot** [comment]
It seems to ignore other settings in VS Code that would be pertinent, such as:
- `css.lint.validProperties` which lets you specify properties not treated as unknown
- `css.lint.emptyRules` which lets you enable or disable warnings for empty rulesets
